src/Fixer/Phpdoc/PhpdocSeparationFixer.php

```diff
@@ -92,6 +129,31 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens): void
         }
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    protected function createConfigurationDefinition(): FixerConfigurationResolverInterface
+    {
+        return new FixerConfigurationResolver([
+            (new FixerOptionBuilder('groups', 'Sets of annotation types to be grouped together.'))
+                ->setAllowedTypes(['string[][]'])
+                ->setDefault(TagComparator::TAG_GROUPS)
+                ->getOption(),
+            (new FixerOptionBuilder('additional_groups', 'Sets of additional annotation types to be grouped together.'))
+                ->setAllowedTypes(['string[][]', 'null'])
+                ->setDefault(self::ADDITIONAL_GROUPS_DEFAULT)
+                ->getOption(),
```

Comment:
```suggestion
            (new FixerOptionBuilder('additional_groups', 'Sets of additional annotation types to be grouped together.'))
                ->setAllowedTypes(['string[][]'])
                ->setDefault([])
                ->getOption(),
```

What's the reason for having both `groups` and `additional groups`? Are `additional groups` supposed to make extending the default configuration easier?

Reply:
Yes, the goal is to make extending the default configuration easier, 

But, off course I can remove ``additional groups`` and use ``groups`` in conjunction with ``TagComparator::DEFAULT_GROUP`` to extend the default configuration.

So, what are your suggestion @jrmajor?

Reply:
I haven't seen this pattern in any other fixer, so I would prefer to leave only the `groups` option. You may wait for other reviewers with removing this though :)

Reply:
I want the style and code to be as consistent as possible, so I'm throwing off the ``additional_groups`` option.